### PR TITLE
ExprMergeComponents | Add delimiter

### PR DESCRIPTION
**Pull request by Fusezion**
Congrates, probably the smallest pr I have planned <3 

This pr aims to add an optional delimiter to the merge components expression while also adding a '-' to the send component effect for command sender

Send component didn't return any known errors but one of my failed builds sent an error at that point which made me aware of this issue.

### CODE REVIEW

File: src/main/java/com/shanebeestudios/skbee/elements/text/effects/EffSendComponent.java [hunk 1]
```diff
@@ -34,7 +34,7 @@ public class EffSendComponent extends Effect {
 
     static {
         Skript.registerEffect(EffSendComponent.class,
-                "send [(text|1¦action[[ ]bar])] component[s] %objects% [to %commandsenders%] [from %-player%]",
+                "send [(text|1¦action[[ ]bar])] component[s] %objects% [to %-commandsenders%] [from %-player%]",
```
Comment (ShaneBeee) [inline src/main/java/com/shanebeestudios/skbee/elements/text/effects/EffSendComponent.java:37]: Why was this added?
Im pretty sure this would cause a console error if you attempt to use this effect in an event where a command sender does not exist.

File: src/main/java/com/shanebeestudios/skbee/elements/text/expressions/ExprMergeComponents.java [hunk 2]
```diff
@@ -17,29 +17,35 @@
...
 
     @SuppressWarnings("NullableProblems")
     @Override
     protected @Nullable BeeComponent[] get(Event event) {
+        if (this.components == null)
+            return null;
         BeeComponent[] components = this.components.getArray(event);
-        return new BeeComponent[]{BeeComponent.fromComponents(components)};
+        String delimiter = this.delimiter.getSingle(event);
+        return new BeeComponent[]{BeeComponent.fromComponents(components, delimiter)};
```
Comment (ShaneBeee) [inline src/main/java/com/shanebeestudios/skbee/elements/text/expressions/ExprMergeComponents.java:48]: Has this been tested without using a delimiter?
I believe this would thru an error when the delimiter is null and you're calling getSingle on it.

File: src/main/java/com/shanebeestudios/skbee/elements/text/effects/EffSendComponent.java (hunk 1 above)
Comment (Fusezion) [reply src/main/java/com/shanebeestudios/skbee/elements/text/effects/EffSendComponent.java:37]: I'll check once I'm home added as I saw it was missing

Might take a few hours as I'm at work for snother 6 hours

File: src/main/java/com/shanebeestudios/skbee/elements/text/expressions/ExprMergeComponents.java (hunk 2 above)
Comment (Fusezion) [reply src/main/java/com/shanebeestudios/skbee/elements/text/expressions/ExprMergeComponents.java:48]: I should have and this should be how a lot of the recipe stuff is and I didn't get any null errors there

**Review by ShaneBeee — APPROVED** [review]
Looks good to me 👍
